dev/builddeps-veloxbe.sh

```diff
@@ -121,6 +122,10 @@ do
         ENABLE_GPU=("${arg#*=}")
         shift # Remove argument name from processing
         ;;
+        --enable_enhanced_feature=*)
```

Comment:
Name suggestion: xxx_features, to imply multiple features are being covered or will be covered. Ditto for other places.